.github/workflows/test.yml

```diff
@@ -11,31 +11,31 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.9, "3.10", 3.11]
       # Run all the matrix jobs, even if one fails.
       fail-fast: false
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: 'poetry'
+      - name: Install mopidy-tidal
+        run: |
+          poetry install
       - name: Lint
         run: |
-          pip install isort black
```

Comment:
So if I understand correctly, poetry completely takes care of these dependencies now?

Reply:
Yes!  They're in the dev group installed by default but not in the build